ui/src/shared/components/HistogramChartBars.tsx

```diff
@@ -219,7 +206,7 @@ class HistogramChartBars extends PureComponent<Props, State> {
       return
     }
 
-    const {data} = hoverGroup
+    const data = _.get(hoverGroup, 'data').reverse()
```

Comment:
I think the `reverse` should be handled by the sorting function that is passed in

Reply:
this was due to the tooltip and the barchart needing flipped sort order